### DISCUSSION TITLE
[Feature] Archive partner assets

### DISCUSSION THREAD

**Discussion by azb-dev** [immich-app/immich#7259] [Feature Request]
### The feature

Templateing only offers 'album' as a way to distinguish between devices/users when uploading. This doesn't work if users have identically named albums and adds uneeded complexity in moving photos from immichs library. So the best and seemingly correct method is to have a user for each user/device you want to store in its own import path e.g.

Library
- deviceA
- userA
- etc

When doing this if you try to archive _UserAs_ photos and you are on _UserBs_ account it informs you you cannot archive another partners assests. 

There is also the case of just wanting to archive another users assests.

I think _UserB_ should have an option in the sharing section, similar to sharing there library with _UserA_, where they can choose if the user they shared with (_UserA_)  can archive their photos (_UserB_).

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #7029
